content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+{{< glossary_definition term_id="eviction" length="all" >}}
+
+During an eviction, the kubelet sets the `PodPhase` for the selected pods to `Failed`.
+This gracefully terminates the pods. 
+
+If the pods are managed by a workload resource (such as {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+or {{< glossary_tooltip text="Deployment" term_id="deployment" >}}) that
+replaces failed pods, the control plane creates new pods in place of the evicted pods.
+
+{{<note>}}
+The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
+before it terminates end-user pods.
+{{</note>}}
+
+When evicting pods, the kubelet respects your configured [PodDisruptionBudget](/docs/tasks/run-application/configure-pdb/).
+You can control which pods are evicted last by setting the [Quality of Service class](/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed)
+on those pods to `Guaranteed`.
+
+The kubelet uses various parameters to make eviction decisions, like the following:
+
+  * Eviction signals
+  * Eviction thresholds
+  * Monitoring intervals
+  * Node conditions
+
+## Triggering eviction
+
+You can configure pod eviction to happen automatically, triggered by a controller. 
+
+You can also request pod eviction manually using the `kubectl drain` command
```

Comment:
This is different from a kubelet eviction.

Reply:
To clarify, this topic is for all types of eviction. The [Safely Drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service) page states that `kubectl drain` evicts pods in a node, so I added that here as one of the possible methods.

Reply:
Nearly all of this page does not apply to draining a node.  The eviction API does graceful deletion (which eviction is not), and _does_ respect pod disruption budgets, for example.  Your pod also won't be set to phase == Failed during a drain.